src/index.js

```diff
@@ -28,7 +30,20 @@ export default declare(({
   SVG_NAME.defaultProps = SVG_DEFAULT_PROPS_CODE;
 `);
 
-  function applyPlugin(importIdentifier, importPath, path, state) {
+  const exportSVG = template(`
+  var SVG_NAME = function SVG_NAME(props) { return SVG_CODE; };
```

Comment:
Is there some way we could invoke buildSvg here, to reuse it, rather than ending up with 4 duplicates of the same code?

Reply:
Im having some trouble with the exporting of a anonymous function but also setting a displayName on it and/or defaultProps :/

Reply:
`var Component = function () {}`?

Reply:
sorry, i was being stupid trying to figure out how that would work if you have multiple `export {default}`s... but then i remembered thats not a thing.